### PR TITLE
Targeting netcoreapp2.2

### PR DESCRIPTION
**Pull request by Anton92nd**
See https://github.com/skbkontur/gremit/issues/14

### CODE REVIEW

**Comment by AndrewKostousov** [comment]
I think, we need to add a test for this `GroboIL.Calli()` method with native calling convention.

**Comment by AndrewKostousov** [comment]
Also after adding netcoreapp2.x targets to Gremit.csproj we now do not have any tests running against netstandard2.0 targeted version.

File: GrEmit/GrEmit.csproj [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <!-- todo (andrew, 21.02.2019): switch netcoreapp2.x targets to netstandard2.1 target when it will be released (see https://devblogs.microsoft.com/dotnet/announcing-net-standard-2-1/) -->
+    <TargetFrameworks>net45;netstandard2.0;netcoreapp2.1;netcoreapp2.2</TargetFrameworks>
```
Comment (AndrewKostousov) [inline GrEmit/GrEmit.csproj:5]: After adding netcoreapp2.x targets to Gremit.csproj we now do not have any tests running against netstandard2.0 targeted version.

File: GrEmit/GroboIL.cs [hunk 2]
```diff
@@ -1996,15 +2000,15 @@ public void Calli(CallingConventions callingConvention, Type returnType, Type[]
...
+#if !NETSTANDARD2_0 // see https://apisof.net/catalog/System.Reflection.Emit.ILGenerator.EmitCalli(OpCode,CallingConvention,Type,Type())
+        /// <summary>
+        ///     Calls the method indicated on the evaluation stack (as a pointer to an entry point) with arguments described by a calling convention.
+        /// </summary>
+        /// <param name="callingConvention">The unmanaged calling convention to be used.</param>
+        /// <param name="returnType">
+        ///     The <see cref="Type">Type</see> of the result.
+        /// </param>
+        /// <param name="parameterTypes">The types of the required arguments to the instruction.</param>
         public void Calli(CallingConvention callingConvention, Type returnType, Type[] parameterTypes)
```
Comment (AndrewKostousov) [inline GrEmit/GroboIL.cs:2011]: I think, we need to add a test for this `GroboIL.Calli()` method with native calling convention.

**Review by AndrewKostousov — CHANGES_REQUESTED** [review]

File: GrEmit/GrEmit.csproj (hunk 1 above)
Comment (AndrewKostousov) [reply GrEmit/GrEmit.csproj:5]: added netcoreapp2.0 target to GrEmit.Tests

File: GrEmit/GroboIL.cs (hunk 2 above)
Comment (Anton92nd) [reply GrEmit/GroboIL.cs:2011]: Done

**Review by AndrewKostousov — APPROVED** [review]

**Pull request merged**